packages/ui/src/variables.css

```diff
@@ -263,3 +263,8 @@
     --question-container-max-width: 53.5rem; /* 856px */
   }
 }
+
+/* CUSTOM */
+:root {
+  --theme-black-opacity-40: #00000040;
```

Comment:
I've just been putting custom ones at the bottom of the definitions above. I don't mind the breakout here if we want to refactor that to separate them. However, if they are specific to a component I've been trying to namespace them. I could see this being named `--modal-side-overlay-background-color`. Thoughts?

Reply:
Right now it's technically specific to this component but down the line if we add more modals in we can just use. 

Also, I do like the idea of breaking it out into a "custom" section to separate them from the "defaults" that shouldn't be adjusted.

Reply:
Cool. Do you want to leave a todo note here for a possible code quality refactor later or capture that in a ticket? Not sure on the best way to handle refactoring what's there already.